services/121-service/src/financial-service-providers/fsp-question.entity.ts

```diff
@@ -24,25 +27,25 @@ export class FspQuestionEntity extends Base121Entity {
 
   @Column('json')
   @ApiProperty({ example: { en: 'label' } })
-  public label: JSON;
+  public label: LocalizedString;
```

Comment:
Great that you got rid of these JSON types! They were really annoying and I felt we always had to jump thourgh hopes to get them working..